doc/elasticdl_on_sqlflow.md

```diff
@@ -0,0 +1,141 @@
+# _Design:_ ElasticDL on sqlflow
+
+## Overview
+
+This is a design doc on integration with [ElasticDL](https://github.com/wangkuiyi/elasticdl).
```

Comment:
Is https://github.com/wangkuiyi/elasticdl a public repo?

Reply:
Not yet.